frontend/src/components/FacilityBooking/ViewBookingCard.tsx

```diff
@@ -0,0 +1,182 @@
+  display: flex;
+  flex-direction: row;
+`
+
+const EventTimingContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const ExitButtonContainer = styled.img`
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+`
+
+const TelegramButtonContainer = styled.img`
+  width: 39px;
+  height: 39px;
+  cursor: pointer;
+`
+
+const StyledText = styled.text<{ fontSize?: string; fontWeight?: string; color?: string }>`
+  font-size: ${(props) => props.fontSize ?? `14px`};
+  color: ${(props) => props.color ?? `#191919`};
+  font-weight: ${(props) => props.fontWeight ?? `400`};
+  text-align: center;
+  line-height: 22px;
+`
+
+type Props = {
+  booking?: Booking | null
+  onClickFunction: Dispatch<SetStateAction<boolean | undefined>>
+}
+
+export const ViewBookingCard = (props: Props) => {
+  const [telegramHandle, setTelegramHandle] = useState<string>()
+  const [bookingStartTimeUnix, setBookingStartTimeUnix] = useState<number>(0)
+  const [bookingEndTimeUnix, setBookingEndTimeUnix] = useState<number>(0)
+
+  useEffect(() => {
+    fetchTelegram(props.booking?.userID)
+    setBookingStartTimeUnix(props.booking?.startTime == undefined ? 0 : props.booking?.startTime)
+    setBookingEndTimeUnix(props.booking?.endTime == undefined ? 0 : props.booking?.endTime)
+  })
+
+  const ExitButton = () => {
+    return <ExitButtonContainer src={ViewBookingCardButton} onClick={() => props.onClickFunction(false)} />
+  }
+
+  const TelegramButton = () => {
+    return <TelegramButtonContainer src={ViewBookingCardUserIcon} />
+  }
+
+  const fetchTelegram = (userID: string | undefined) => {
+    if (userID) {
+      try {
+        fetch(DOMAIN_URL.FACILITY + ENDPOINTS.TELEGRAM_HANDLE + '/' + userID, {
+          method: 'GET',
+          mode: 'cors',
+        })
+          .then((resp) => resp.json())
+          .then((data) => {
+            if (data.data === '' || data.data === undefined) {
+              throw data.err
+            } else {
+              setTelegramHandle(data.data)
+            }
+          })
+      } catch (err) {
+        console.log(err)
+        setTelegramHandle('')
+      }
+    }
+  }
+
+  return (
+    <BackgroundOverlay>
+      <BookingContainer>
+        <BookingHeader>
+          <StyledText fontSize="27px" fontWeight="700">
+            {props.booking?.eventName}
+          </StyledText>
+          <StyledText fontSize="14px" fontWeight="400" color="#666666">
+            {props.booking?.ccaName}
+          </StyledText>
+        </BookingHeader>
+        <TelegramDetailsContainer onClick={telegramHandle ? () => openUserTelegram(telegramHandle) : undefined}>
+          <TelegramButton />
+          <StyledText fontSize="14px" fontWeight="400">
+            @{telegramHandle}
+          </StyledText>
+        </TelegramDetailsContainer>
+        <EventDetailsContainer>
+          <EventTimingContainer>
+            <StyledText color="#666666">{unixToFullDay(bookingStartTimeUnix)}</StyledText>
```

Comment:
use boolean param instead of explicitely passing in the colour since only 2 colours are used 
e.g. `<StyledText grayed>`